pkg/cli/admin/upgrade/status/status.go

```diff
@@ -240,6 +264,20 @@ func (o *options) Run(ctx context.Context) error {
 		fmt.Fprintf(o.ErrOut, "warning: No current %s info, see `oc describe clusterversion` for more details.\n", clusterStatusFailing)
 	}
 
+	var alertData AlertData
+
+	alertBytes, err := inspectalerts.GetAlerts(ctx, o.getRoute, o.RESTConfig.BearerToken)
+	if err != nil {
+		fmt.Println("Unable to fetch alerts from thanos, ignoring alerts in 'Update Health': ", err)
+	}
+
+	// Unmarshal the JSON data into the struct
+	if err := json.Unmarshal(alertBytes, &alertData); err != nil {
+		return fmt.Errorf("unmarshaling alerts: %w", err)
+	}
+
+	updateInsights = append(updateInsights, parseAlertDataToInsights(alertData, startedAt)...)
```

Comment:
These two should be conditional only when GetAlerts was successful.

`parseAlertDataToInsights` should be unit-tested